.github/workflows/PROJECT-AUTO-CHANGELOG-CONTROL.yaml

```
@@ -309,15 +309,23 @@ jobs:
-        env:
-          GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
         run: |
-          echo "🚀 배포 워크플로우 트리거 중..."
-          gh workflow run "PUBLISH-SUH-NEXUS-REPOSITORY"
-          echo "✅ 배포 워크플로우가 트리거되었습니다!"
+          echo "🚀 Deploy 브랜치로 강제 머지 시작..."
+          
+          # deploy 브랜치 체크아웃
+          git fetch origin deploy:deploy
+          git checkout deploy
+          
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)